mongodb.cpp

```diff
@@ -252,6 +255,10 @@ static class MongoDBExtension : public Extension {
 
 			loadSystemlib("mongodb");
 			mongoc_init();
+
+			std::string s = "HHVM "; s += HHVM_VERSION;
+			mongoc_handshake_data_append("Hippo", HIPPO_VERSION, s.c_str());
```

Comment:
Let's replace "Hippo" with "ext-mongodb:HHVM". I'd rather not publicize the Hippo/Phongo names, and this will allow folks to easily differentiate between HHVM and PHP drivers by just the `driver.name` field (if they don't notice `platform`).